freezegun/api.py

```diff
@@ -66,7 +66,7 @@ def _setup_modules_cache():
     # FIXME: move this definition to be at the top-level
     real_time_object_ids = set(id(obj) for obj in real_date_objects)
     result = {}
-    for mod_name, module in sys.modules.items():
+    for mod_name, module in list(sys.modules.items()):
```

Comment:
it's unnecessary - if `sys.modules` is dict , then `sys.modules.items()` is iterable containing pairs of key and value in both py2 and py3  
casting it to `list` will not break it, but slow down a bit

Reply:
Without creating the copy, I'm occasionally getting a `RuntimeError: dictionary changed size during iteration`, which makes sense since additional modules can get loaded.

Ideas?

Reply:
I would assume that the size of that collection is small enough so that casting to list once does not cause a relevant slowdown.

Reply:
so commit message is misleading

Reply:
No, the PR title is about an architectural optimization that saves seconds of run time, not microoptimizations that make a difference of a few microseconds.

Reply:
I meant this particular commit, not entire PullRequest  
Reading `Make sys.modules.items() compatible with python 3` I expect some py3 adjustment, but I get runtime error prevention.  
That was my point, sir :)

Reply:
The commit message is not misleading. In python 2, `items()` returns a list (copy), which is what is needed here. But in python 3 it returns an iterator, which causes faulty behavior. So it is casted to list for python 3 support.

Reply:
+1